include/Metabox/Base.php

```diff
@@ -301,4 +301,41 @@ protected function render_result( $posts_moved ) {
 			'updated'
 		);
 	}
+
+	protected function render_roles_dropdown( $name = '', $hide_no_role_option = false ) {
+		global $wp_roles;
+		$roles = $wp_roles->roles;
+
+		if ( ! ( $users = wp_cache_get( 'bm_users', 'bulk-wp' ) ) ) {
+			$users = get_users();
+			wp_cache_add( 'bm_users', 'bulk-wp' );
+		}
+
+		$bulk_move = bulk_move();
+
+		$users_by_roles = $bulk_move->helper->get_users_count_by_roles( $users, $roles );
+
+		$field_name  = $this->meta_box_slug;
+		$field_name .= ! empty( $name ) ? '-' . $name : $name;
+		$field_name .= '-roles-list';
+
+		?>
+		<select id="<?php echo $field_name; ?>"
+				name="<?php echo $field_name; ?>">
+			<?php if ( ! $hide_no_role_option ) : ?>
+				<option value="norole">No role <?php echo '(' . $users_by_roles['norole'] . ')' ?></option>
```

Comment:
This string should be made available for translation.

Reply:
> This string should be made available for translation.

Fixed